### DISCUSSION TITLE
Did Firefly originally have the 24 word seedphrase?

### DISCUSSION THREAD

**Discussion by AmericanNDN** [iotaledger/firefly#8183] [Help & Support] [answered]
I created a Firefly account 2-3 years ago to hold my IOTA.  Due to a vehicle accident I didn't keep up with it for a couple years.  I am now trying to withdraw my Iota and cannot find .kdbx file or stronghold file.  I never was given a 24 word seed phrase.  Is there any hope?

**Comment by ashnik** [comment]
Yes

**Comment by begonaalvarezd** [comment, marked as answer]
> I created a Firefly account 2-3 years ago to hold my IOTA. Due to a vehicle accident I didn't keep up with it for a couple years. I am now trying to withdraw my Iota and cannot find .kdbx file or stronghold file. I never was given a 24 word seed phrase. Is there any hope?

If you have a 81char seed instead of a 24 words mnemonic, you should migrate using the legacy migration tool https://github.com/iotaledger/legacy-migration-tool
